Team 6 Senior Project/CSVExporter.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Team_6_Senior_Project
+{
+    internal class CSVExporter
+    {
+        private static void ExportTOCSV(string data)
+        {
+            SaveFileDialog save = new()
+            {
+                Filter = "CSV Files | *.csv",
+                Title = "Save your table to a csv file"
+            };
+            save.ShowDialog();
+            if (save.FileName != null)
+            {
+                try
+                {
+                    StreamWriter sw = new(save.FileName);
+                    sw.Write(data);
+                    sw.Close();
+                }
+                catch (Exception ex)
+                {
+                    MessageBox.Show("An error has occured while trying to save your file.  Please try again");
+                    Console.WriteLine(ex.Message);
+                }
+            }
+        }
```

Comment:
close the SaveFileDialog with a dispose after your if. Between lines 32 and 33:
save.Dispose();

Reply:
Went ahead and added save.Dispose() after the if statement.  I changed up the conditions for the if statement so it wouldn't display an error whenever you clicked cancel or exited out without saving on the save window.

Reply:
I was just staring at that if trying to figure it out.